src/frequenz/client/common/pagination/__init__.py

```diff
@@ -0,0 +1,87 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Module to define the pagination used with the common client."""
+
+from __future__ import annotations  # required for constructor type hinting
+
+from dataclasses import dataclass
+from typing import Self
+
+# pylint: disable=no-name-in-module
+from frequenz.api.common.v1.pagination_info_pb2 import Info as PBPaginationInfo
+from frequenz.api.common.v1.pagination_params_pb2 import Params as PBPaginationParams
```

Comment:
The imports seem to be messed up. It should be:

```suggestion
from frequenz.api.common.v1.pagination.pagination_info_pb2 import PaginationInfo
from frequenz.api.common.v1.pagination.pagination_params_pb2 import PaginationParams
```